homework_unit_11/main.js

```diff
@@ -0,0 +1,61 @@
+// Example 1
+// option 1
+const result = []
+const loopArray = (someThing, count) => {
```

Comment:
```suggestion
const dublicate = (word, times) => {
```
Đặt tên cho dễ hiểu và sát. với nội dung 1 tí